core/camel-core-model/src/main/java/org/apache/camel/model/SplitDefinition.java

```diff
@@ -517,6 +520,14 @@ public void setExpression(ExpressionDefinition expression) {
         super.setExpression(expression);
     }
 
+    public String getDelimiter() {
+        return delimiter;
+    }
+
+    public void setDelimiter(String delimiter) {
```

Comment:
We need documentation for this new option. See elsewhere in this class for those fluent methods that have javadoc. Then add a fluent method for delimiter with javadoc

Reply:
Thanks Claus - see latest commit 50d57a986c6fe748313a8d195060e74e4cb3fc51
I wasn't sure how to address the build failure reported with the previous commit _(sorry, I didn't grab the error msg prior to pushing the latest commit)_. A new build is in progress now. If it fails, I may need some pointers on how to resolve.
Thanks in advance